### PR TITLE
Imviz: Show coordinates in degrees too

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to address the first point mentioned in #598 .

[🐱](https://jira.stsci.edu/browse/JDAT-1930)

@jenneh, would be nice to have your input on this. Thanks!

### Todo

- [x] Implement https://github.com/spacetelescope/jdaviz/pull/971#issuecomment-977221303
- [x] Merge pllim/jdaviz#4

### Screenshots

*Without* this patch (HST/ACS):

Screenshot 2021-11-23 150703

*With* this patch:

| Data source | Screenshot |
| --- | --- |
| HST/ACS | Screenshot 2021-11-24 012013 |
| HST/ACS (out of bounds) | Screenshot 2021-11-24 012714 |
| JWST/NIRCam | Screenshot 2021-11-24 011923 |
| JWST/NIRCam (out of bounds) | Screenshot 2021-11-24 012038 |
| Numpy array | Screenshot 2021-11-24 011948 |

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set? Milestone is only currently required for PRs related to Imviz MVP.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: jdaviz/configs/imviz/plugins/coords_info/coords_info.vue [hunk 1]
```diff
@@ -2,5 +2,6 @@
     <div v-if="pixel" style="white-space: nowrap;">
         <b v-if="pixel">Pixel </b>{{ pixel }}&nbsp;&nbsp;<b v-if="value">Value </b>{{ value }}<br>
         <b v-if="world">World </b>{{ world }}<br>
+        <b v-if="world">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</b>{{ world_deg }}<br>
```
Comment (pllim) [inline jdaviz/configs/imviz/plugins/coords_info/coords_info.vue:5]: Maybe @kecnry knows a better way to do this... 😅 

Kinda hard to get them align perfectly from image to image, but hopefully this is good enough without having to specify a special `div` for each sub-element.

Comment (rosteen) [reply jdaviz/configs/imviz/plugins/coords_info/coords_info.vue:5]: You should be able to use here I think, to make everything line up without this workaround.

**Comment by rosteen** [comment]
I was going to say that I dislike how tall the toolbar is getting to fit this and recommend a toggle to switch between the two types, but then I saw that the toolbar seems to be just as tall on main. So I guess it's fine!

File: jdaviz/configs/imviz/plugins/coords_info/coords_info.vue (hunk 1 above)
Comment (kecnry) [reply jdaviz/configs/imviz/plugins/coords_info/coords_info.vue:5]: I can think of many *other* ways to do this... but they all require nesting more elements or providing a fixed width for your empty `<b>` element.  But I think unless we want to structure this further, this is probably fine for now.

**Review by kecnry — APPROVED** [review]
Seems straightforward enough to me and confirmed to work and display well on my end!

File: jdaviz/configs/imviz/plugins/coords_info/coords_info.vue (hunk 1 above)
Comment (pllim) [reply jdaviz/configs/imviz/plugins/coords_info/coords_info.vue:5]: So should I try out the `<v-col>` stuff, or leave it as `&nbsp;` like it is 1999?

Comment (kecnry) [reply jdaviz/configs/imviz/plugins/coords_info/coords_info.vue:5]: I guess using columns would be a little more font-safe (for non-fixed-width fonts where nbsp might not line up well) and would be more future proof to maintain down the road

**Comment by PatrickOgle** [comment]
The mouseover coordinates layout shown above looks good.  Official review later...

**Comment by PatrickOgle** [comment]
Way too many significant figures for the decimal degree coordinates.   I think six decimal places will cover most needs (microarcsec).

**Comment by pllim** [comment]
The significant digits were mostly to align the numbers. 😆 Let me see if `<v-col>` would work better...

**Comment by pllim** [comment]
💭 `<v-col>` won't work as-is. I'll have to write a new class to internally separate the RA and Dec portion of the display for things to align properly next week.

🦃

File: jdaviz/configs/imviz/plugins/coords_info/coords_info.vue [hunk 2]
```diff
@@ -1,6 +1,23 @@
 <template>
-    <div v-if="pixel" style="white-space: nowrap;">
-        <b v-if="pixel">Pixel </b>{{ pixel }}&nbsp;&nbsp;<b v-if="value">Value </b>{{ value }}<br>
-        <b v-if="world">World </b>{{ world }}<br>
-    </div>
+  <div v-if="pixel" style="white-space: nowrap;">
+    <table>
```
Comment (pllim) [inline jdaviz/configs/imviz/plugins/coords_info/coords_info.vue:3]: My vuejs-fu is too weak. I think this accomplishes what you asked for, but I don't know how robust it is across browser and monitor settings...

Comment (kecnry) [reply jdaviz/configs/imviz/plugins/coords_info/coords_info.vue:3]: Looks good to me.  At small widths, it overlaps the plugin menu button, but that button isn't clickable while mousing over the image anyways.  We have another ticket to eventually deal with minimum widths and overflowing across the app, so I don't think you need to worry about that too much right now.

**Review by PatrickOgle — APPROVED** [review]
Looks good.

**Comment by pllim** [comment]
Kyle has fixed the padding problem, so I am going to merge when CI passes. Thanks, everyone!

**Pull request merged**